File: ios/WrapperOld/WrapperOld.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(WrapperOld, NSObject)

RCT_EXTERN_METHOD(multiply:(float)a withB:(float)b
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)
@end

File: ios/WrapperOld/WrapperOld.swift
import ObjectiveC
import Foundation
import libxcode

@objc(WrapperOld)
class WrapperOld: NSObject {

  @objc
  func multiply(
    _ a: Float,
    withB b: Float,
    withResolver resolve: RCTPromiseResolveBlock,
    withRejecter reject: RCTPromiseRejectBlock
  ) -> Void {
      let math = Math()
      math.greet()
      resolve(math.multiply(a: a, b: b))
  }
}
